### ISSUE TITLE
how we add decives in vs ode

### ISSUE THREAD

**Issue by Mazhar012** [microsoft/vscode-edge-devtools#1551]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like:**

**Describe alternatives you've considered:**

**Additional context:**

**Comment by captainbrosset** [comment]
I don't think you can add your own custom devices in the VSCode DevTools extension right now.
@vidorteg is this something you're tracking already?

**Comment by codepo8** [comment]
I think the extension inherits the list from Edge, but there is no way to edit it. Might be good to have a setting for that.

**Comment by Mazhar012** [comment]
How

**Comment by dsine-de** [comment]
This would be a great feature because currently there is no way to keep a aspect ratio of i.e. 16:9 or a fixed viewport size of 1920x1080. If the dropdown is set to "Responsive", the values change as soon as you resize the devtools panel.

**Comment by KotKarol** [comment]
Has anyone found a workaround for this issue? I couldn't find any chromium launch options that would help.
I develop mainly on Smart TVs so i would like to toggle between 720p and 1080p predefined viewport, and not be forced to resize it when i change the vs code layout.

Currently the only workaround for me that is good enough is to set the device to "Laptop with MDPI screen", which keeps the ratio.

**Comment by captainbrosset** [comment]
Unfortunately this has not been added to our extension yet.
